do-dont/doc/do-dont.md

```diff
@@ -124,15 +129,51 @@ const customIcon = svg`<svg
   />
 </svg>`;
 
-export const do_custom = () => html` <dockit-do
-  .color=${'#fcba03'}
-  .icon=${customIcon}
-  .label=${'HINTS'}
+export const do_custom = () => `<dockit-do
+  style="--dockit-do-color: #fcba03"
+  label="HINTS"
 >
+  ${customIcon}
   <ul slot="instructions">
     <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
     <li>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris</li>
     <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
   </ul>
 </dockit-do>`;
 ```
+
+### Themed component
+
+`dockit-do` and `dockit-dont` support theming and custom styling via `::part()` selector, see example:
+
+```html preview-story
+<style>
+  dockit-do.themed-do {
+    --dockit-do-color: #000080;
+  }
+  dockit-do.themed-do::part(container) {
+    border-style: solid;
+    border-width: 2px;
+    border-color: #000080;
+    border-radius: 1rem;
+  }
+  dockit-do.themed-do::part(component-container) {
+    background-color: #00008020;
+  }
+  dockit-do.themed-do::part(instructions-container) {
```

Comment:
earlier I suggested to simplify CSS selectors and stick to classes
here it remained as it was before
is it because you expect `::part` to be only working with such selectors?
why is `.themed-do` not sufficient?

Reply:
it works, it's something I missed, fixing